wrangler-core/src/main/java/co/cask/directives/aggregates/DefaultTransientStore.java

```diff
@@ -41,12 +43,23 @@
    * @param value associated with the variable.
    */
   @Override
-  public void increment(String name, long value) {
-    Long count = get(name);
+  public void increment(TransientVariableScope scope, String name, long value) {
+    if(scope == TransientVariableScope.GLOBAL) {
```

Comment:
space after if. otherwise will fail checkstyle. Would be good to add checkstyle from cdap repo if you don't have it configured already.